source/_redirects

```diff
@@ -76,6 +76,7 @@
 /developers/rest_api https://developers.home-assistant.io/docs/api/rest/
 /developers/server_sent_events https://developers.home-assistant.io/docs/external_api_server_sent_events/
 /developers/websocket_api https://developers.home-assistant.io/docs/api/websocket/
+/developers/code_of_conduct/ /code_of_conduct
```

Comment:
For the `_redirects` file, this last slash of the URL to redirect, can be removed. Both `/developers/code_of_conduct/` and `/developers/code_of_conduct` will work at that point.

```suggestion
/developers/code_of_conduct /code_of_conduct
```

This applies to all other redirects added as well.